### ISSUE TITLE
Can't find the labels for the comments

### ISSUE THREAD

**Issue by MiladAlshomary** [tuhinjubcse/IMHO-NAACL2019#1]
Hi,

Again, I came across your work! Actually being able to mine claims from text suffices in my case, so I will train a model according to this paper. I was able to access the comments_text file from the bitbucket project, but I couldn't find the labels for the sentences (claim/not-claim). Could you please point me to where I can find that?

Thanks for your help!

**Comment by tuhinjubcse** [comment]
Hi
Milad

There is no claim / non-claim label. These are a bunch of self labeled claims. We hypothesis fine-tuning a language model on a bunch of self-labeled opinions makes ur discern between the languages people use as a claim. Also can you please star the repos both AMPERSAND and t
this if you find them useful?

**Comment by MiladAlshomary** [comment]
Thanks for the fast reply!

I see, so basically the classification model was trained on other argumentation datasets (according to the paper). That is what  in the train.tsv and test.tsv files that are loaded in claim_classifier.py. Could you please also provide these files?

Thanks,

**Comment by tuhinjubcse** [comment]
Hi
Milad

I uploaded

for MT , PE , WD there are 10 files , basically 10 instances of train dev-test based on random splits
you can use any of the cross validations  or use all and average

These are different domains , so I recommend based on your claim extraction purpose use the dataset that best fits your case

MT , PE are essays
WD ,CMV are dialogues/discussion forums

For CMV , the whole file claimnonclaim.txt is given 
Feel free to split it into train dev test

**Comment by MiladAlshomary** [comment]
Thanks a lot for sharing!
